### DISCUSSION TITLE
[FreeBSD 13-STABLE] self compiled vaultwarden - log mess

### DISCUSSION THREAD

**Discussion by tinola** [dani-garcia/vaultwarden#2352] [General]
Hi,

I compiled vaultwarden from master branch yesterday (no docker).
Unfortunately it started to show some mess (color codes?) in syslog while getting icons:

`Mar  3 10:15:22 vw vaultwarden[9206]: [2022-03-03 10:15:22.394][request][INFO] GET /icons/some.tld/icon.png`
`Mar  3 10:15:22 vw vaultwarden[9206]: [2022-03-03 10:15:22.394][response][INFO] ^[[36m(^[[0m^[[37micon_internal^[[0m^[[36m)^[[0m ^[[32mGET^[[0m ^[[4;34m/icons^[[0m^[[34m/<domain>/icon.png^[[0m => 200 OK
`

Earlier there was not such mess. Is there any possibility to turn off it?

**Comment by BlackDex** [comment]
Try setting `ROCKET_CLI_COLORS=0` as an env

**Reply by tinola** [reply]
@BlackDex, thank you very much! That did the trick :)

For others:

Edit file **/usr/local/etc/rc.conf.d/vaultwarden** put those lines and save:

`# disable cli_colors mess in syslog`
`ROCKET_CLI_COLORS=0`
`export ROCKET_CLI_COLORS`

Finally restart vaultwarden daemon.
